sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -46,9 +46,9 @@ grammar SqlBase;
   }
 
   /**
-   * When true, ANSI SQL parsing mode is enabled.
+   * When true, use ANSI SQL standard keywords.
    */
-  public boolean ansi = false;
+  public boolean use_SQL_standard_keywords = false;
```

Comment:
nit: The name looks a little weird to me. This value defines not keywords but behaviours? So, `follow_SQL_standard_behaviours `?

Reply:
Thanks for the suggestion, how about keep both `keyword` and `behavior` in this flag, cause for the comment here, it is still closely related with keywords?
https://github.com/apache/spark/blob/5a70af7a6c271e12a1f8b508dcede804e30a1b75/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4#L954-L960

Reply:
yea, looks fine, thanks!